src/components/Partners/EnablingPartners.js

```diff
@@ -0,0 +1,77 @@
+import { A } from "@commons-ui/core";
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  root: {
+    paddingTop: typography.pxToRem(56.69),
+  },
+  title: {
+    fontWeight: 900,
+    color: palette.grey.dark,
+    marginBottom: typography.pxToRem(49.38),
+  },
+  link: {
+    display: "flex",
+    justifyContent: "center",
+    "&>div": {
+      filter: "grayscale(1)",
+      "&:hover": {
+        filter: "unset",
+        boxShadow: "0px 3px 6px #00000029",
+      },
+    },
+  },
+  logo: {
+    margin: "0 auto ",
+  },
+}));
+function EnablingPartners({ title, partners, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <Grid container xs={12}>
+        <Grid xs={12}>
```

Comment:
```suggestion
        <Grid item xs={12}>
```
Failed prop type: The prop `xs` of `Grid` must be used on `item`.